java_training/src/com/lftechnology/jan12/ArrayListMain.java

```diff
@@ -0,0 +1,94 @@
+package com.lftechnology.jan12;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * This class explores the functionalities of the {@link ArrayListManipulator} class
+ * 
+ * @author nerisa
+ * 
+ */
+public class ArrayListMain {
+	private static final Logger LOG = Logger.getLogger(ArrayListMain.class.getName());
+
+	public static void main(String[] args) {
+		ArrayListManipulator mainList = new ArrayListManipulator();
+		for (int i = 1; i < 50; i++) {
+			mainList.addElement(i);
+		}
+		LOG.log(Level.INFO, "Main list:");
+		mainList.display();
+		ArrayListManipulator subList = mainList.shuffleAndRetrieve(0, 6);
+		LOG.log(Level.INFO, "Sub list:");
+		subList.display();
+		subList.sort();
+		LOG.log(Level.INFO, "Sorted sub list:");
+		subList.display();
+	}
+}
+
+/**
+ * This class explores the {@link ArrayList} class has methods to
+ * <ol>
+ * <li>Add elements to its instance variable</li>
+ * <li>Shuffle the elements of the instance variable and retrieve a specified length of sublist</li>
+ * <li>Sort the instance variable</li>
+ * <li>Display the instance variable</li>
+ * </ol>
+ * 
+ * @author nerisa
+ * 
+ */
+class ArrayListManipulator {
+	public List<Integer> list = new ArrayList<Integer>();
```

Comment:
See here, Is this member variable exposed to other class, or is this necessary to expose to main class, if not please make this private. Since you are adding elements to this list using addElement() method , so no need to expose this variable